Apps/phdigicert/digicert.json

```diff
@@ -0,0 +1,802 @@
+{
+    "appid": "86fd2d88-1b9c-4dbb-b6aa-1fec6f771c0f",
+    "name": "DigiCert",
+    "description": "This app uses the DigiCert automation API to manage ssl certificates",
+    "type": "devops",
+    "product_vendor": "DigiCert",
+    "product_name": "DigiCert Services API",
+    "logo": "digicert.svg",
+    "logo_dark": "digicert_dark.svg",
+    "product_version_regex": ".*",
+    "python_version": "3",
+    "publisher": "Splunk Community",
+    "license": "Copyright (c) 2021 Splunk Inc.",
+    "app_version": "2.0.0",
```

Comment:
If this is an Initial release, version should be 1.0.0

Reply:
@dgopani-crest done with the said changes